config.yml

```diff
@@ -1096,10 +1096,43 @@ nodes:
     fields:
       - name: parent
         type: node?
+        comment: |
+          The left-hand node of the path, if present. Can be `nil` or any kind
+          of a node returning a non-void value. Will be missing when constant
+          lookup is at the root of the module tree (i.e. `::Foo` has no
+          `parent`).
+
+              Foo::Bar
+              ^^^
+
+              self::Test
+              ^^^^
+
+              a.b::C
+              ^^^
       - name: child
         type: node
+        comment: |
+          The right-hand node of the path. Always a `ConstantReadNode`.
```

Comment:
The [Advent of Prism section](https://kddnewton.com/2023/12/03/advent-of-prism-part-3.html#constantpathnode) states that `child` is "almost always a constant read". 

Looking at the calls to `pm_constant_path_node_create` in `src/prism.c`, they're all passed the result of `pm_constant_read_node_create`, with the exception of [this one](https://github.com/ruby/prism/blob/a81b4ce9cbeda7118461beb78d10564c11c6319b/src/prism.c#L17020) which is passed a `MissingNode` & results in an error.